pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "landingai"
-version = "0.0.50"
+version = "1.0.0"
```

Comment:
I would stay on v0 for longer to not give the impression that the library is production ready... how about v0.1.0?

Reply:
done